bumblebee/BumblebeeSource.cpp

```diff
@@ -0,0 +1,39 @@
+#include "BumblebeeSource.h"
+
+#include <iostream>
+
+
+bool is_prime(int number) {
```

Comment:
Do you want to receive negative numbers?
Consider enforcing only non-negative are received by this function at compile time (via a proper type)